### PR TITLE
TPI-3217: Remove CC Installment

### PR DESCRIPTION
**Pull request by candrasaputra**
## Description

`put the description of your PR here`

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Cosmetic change (text changing or color changing)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code (if necessary), particularly in hard-to-understand areas
- [x] My changes generate no new vulnerabilities
- [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshot

`put screenshot here if there are any ui changes`

### CODE REVIEW

**Review by IreneGohtami — APPROVED** [review]

**Comment by hakiemaul** [comment]
@candrasaputra , is it possible for this change to also still support `installment_info` functionality? the one that is requested by Kanmo. I think we forgot about it during our grooming haha

**Review by yanoandri — APPROVED** [review]

File: Xendit/M2Invoice/Model/Payment/CCHosted.php [hunk 1]
```diff
@@ -62,7 +62,7 @@ public function isAvailable(\Magento\Quote\Api\Data\CartInterface $quote = null)
 
         $cardPaymentType = $this->dataHelper->getCardPaymentType();
 
-        if (($cardPaymentType === 'popup' && $this->methodCode === 'CCHOSTED') || $this->methodCode === 'CC_INSTALLMENT' || $this->methodCode === 'CC_SUBSCRIPTION') {
+        if (($cardPaymentType === 'popup' && $this->methodCode === 'CCHOSTED') || $this->methodCode === 'CC_SUBSCRIPTION') {
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/Model/Payment/CCHosted.php:65]: do we still need cchosted?

File: Xendit/M2Invoice/etc/payment.xml [hunk 2]
```diff
@@ -33,9 +33,6 @@ https://github.com/magento/magento2/blob/2.2.0-rc2.1/app/code/Magento/Multishipp
         <method name='cchosted'>
             <allow_multiple_address>1</allow_multiple_address>
         </method>
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/etc/payment.xml:35]: we can remove cchosted

File: Xendit/M2Invoice/view/frontend/web/js/view/payment/method-renderer.js [hunk 3]
```diff
@@ -54,10 +54,6 @@ define(
                 type: 'cchosted',
                 component: 'Xendit_M2Invoice/js/view/payment/method-renderer/cchosted'
             },
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/view/frontend/web/js/view/payment/method-renderer.js:56]: we can remove cchosted

File: Xendit/M2Invoice/view/frontend/layout/multishipping_checkout_billing.xml [hunk 4]
```diff
@@ -13,7 +13,6 @@
                 <argument name="form_templates" xsi:type="array">
                     <item name="cc" xsi:type="string">Xendit_M2Invoice::multishipping/cc.phtml</item>
                     <item name="cchosted" xsi:type="string">Xendit_M2Invoice::multishipping/cchosted.phtml</item>
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/view/frontend/layout/multishipping_checkout_billing.xml]: can remove cchosted

File: Xendit/M2Invoice/view/frontend/web/js/view/payment/method-renderer.js (hunk 3 above)
Comment (IreneGohtami) [reply Xendit/M2Invoice/view/frontend/web/js/view/payment/method-renderer.js:56]: ini ya can, satu lagi ketinggalan. hehe @candrasaputra

**Pull request merged**